corda-common/src/test/java/ch/cordalo/corda/common/states/CordaloLinearStateTest.java

```diff
@@ -0,0 +1,107 @@
+package ch.cordalo.corda.common.states;
+
+import net.corda.core.contracts.UniqueIdentifier;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Test;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.notNullValue;
+import static org.junit.Assert.assertThat;
+
+public class CordaloLinearStateTest {
+
+    private class UnderTest extends CordaloLinearState {
+
+        public UnderTest(UniqueIdentifier linearId) {
+            super(linearId);
+        }
+
+        public UnderTest(String externalId) {
+            super(externalId);
+        }
+
+        public UnderTest() {
+        }
+
+        @NotNull
+        @Override
+        protected Parties getParties() {
+            return null;
```

Comment:
Never return null here. 
We should validate this in the get participants method.

Reply:
it is a delegate of the class under test (a stub), if i code something here, it make also no sense to test it as the implementation is dummy.
I change it and add a unit test but it test nothing...